en/releases/core_components.md

```diff
@@ -0,0 +1,55 @@
+# PX4 Core Components
+
+These core components are *guaranteed* to work with a stable point release of PX4. They form the foundation of the PX4-Autopilot and it's core functionality.
+
+## Core component criteria
+
+Each core component must have:
+- Maintainer
+- Tester
+
+## PX4 Core Components List
+
+| Core component (category) | Sub component | Maintainer | Tester |
+|---|---|---|---|
+| uORB (Architecture) |  |  |  |
+| Parameters (Architecture) |  |  |  |
+| Realtime Clock (Architecture) |  |  |  |
+| Work queue (Architecture) |  |  |  |
+| GNSS (State Estimation) | Velocity / Position |  |  |
+|  | Heading |  |  |
+| Optical flow (State Estimation) |  |  |  |
+| External vision (State Estimation) |  |  |  |
+| Boards (NuttX) | Pixhawk standard |  |  |
+| IMU (Driver) |  |  |  |
+| Distance Sensor (Driver) |  |  |  |
+| Failure Injection (Driver) |  |  |  |
```

Comment:
I think it would make sense to add the RTK GPS - Heading estimation from dual GPS as supported by you @AlexKlimaj, right?

Reply:
Yes

Reply:
Done, thanks!